res/values/strings_en.arb

```diff
@@ -722,6 +722,10 @@
   "do_not_have_enough_gas_asset": "You do not have enough ${currency} to make a transaction with the current blockchain network conditions. You need more ${currency} to pay blockchain network fees, even if you are sending a different asset.",
   "totp_auth_url": "TOTP AUTH URL",
   "awaitDAppProcessing": "Kindly wait for the dApp to finish processing.",
+  "normal_exchange_description": "Changenow, SideShift, SimpleSwap, and Trocador",
+  "moonpay_exchange_description": "Seamless cryptocurrency swapping.\\nDocuments required",
```

Comment:
Please change `Documents required` to `Account and KYC required`
also please check the generated translation because the `\n` got messed up in a lot of localization files